nxcomp/configure.ac

```diff
@@ -39,6 +35,43 @@ m4_ifdef([PKG_INSTALLDIR], [PKG_INSTALLDIR],
                        [],[with_pkgconfigdir='$(libdir)/pkgconfig'])
           AC_SUBST([pkgconfigdir], [${with_pkgconfigdir}])])
 
+AC_PATH_PROG(PKGCONF, pkg-config)
```

Comment:
@Ionic: wow, this works like sharm. Tested with three occasions:

  * no libjpeg-dev installed at all
  * libjpeg-dev installed (including .pc file)
  * libjpeg-dev installed, but .pc file renamed